app/pb_kits/playbook/pb_bar_graph/bar_graph.rb

```diff
@@ -0,0 +1,109 @@
+module Playbook
+  module PbBarGraph 
+    class BarGraph < Playbook::PbKit::Base
+      PROPS = [
+          :configured_classname,
+          :configured_chart_data,
+          :configured_title,
+          :configured_subtitle,
+          :configured_axis_title,
+          :configured_point_start,
+          :configured_orientation].freeze
+
+      def initialize(
+        chart_data: default_configuration,
+        title: default_configuration,
+        subtitle: default_configuration,
+        axis_title: default_configuration,
+        point_start: default_configuration,
+        classname: default_configuration,
+        orientation: default_configuration
+      )
+        self.configured_classname = classname
+        self.configured_chart_data = chart_data
+        self.configured_title = title
+        self.configured_subtitle = subtitle
+        self.configured_axis_title = axis_title
+        self.configured_point_start = point_start
+        self.configured_orientation = orientation
+      end
+
+      def orientation 
+        orientation_options = %w(vertical horizontal)
+        one_of_value(configured_orientation, orientation_options, "vertical")
+      end
+
+      def chart_type
+        case self.orientation 
+        when "vertical"
+          "column"
+        when "horizontal"
+          "bar" 
+        else
+          "column"
+        end
+      end
+
+      def title
```

Comment:
All of these could utilize the helpers to be more condensed if you want to look into the default_value helper
```
def title
  default_value(configured_title, "")
end
```